lib/asciidoctor/js/asciidoctor_ext/browser/open_uri.rb

```diff
@@ -0,0 +1,11 @@
+module OpenURI
+  def OpenURI.open_uri(name, *rest)
```

Comment:
This part assumes that Asciidoctor core will only use `open_uri` when reading a text file from an URI.
Please note that Asciidoctor core uses `open_uri` to read images in `generate_data_uri_from_uri` but this method is overridden in Asciidoctor.js.